FitNesseRoot/DbFit/AcceptanceTests/JavaTests/InformixTests/FlowMode/InformixMode/SetUp/content.txt

```diff
@@ -0,0 +1,3 @@
+!|dbfit.InformixTest|
+
+|Connect|localhost:16466|dbfit|DBFIT|dbfit_ifx|
```

Comment:
Actually for standalone we're already using `ConnectUsingFile` as far as I see...